src/config/Config.h

```diff
@@ -47,5 +54,6 @@ class Config {
     nlohmann::json mNestedJson;
     std::string mPath;
     bool mIsNewInstance;
+    std::vector<std::shared_ptr<ConfigVersionUpdater>> mVersions;
```

Comment:
same comment as before about naming, i like `mVersionUpdaters` but i'm open to other ideas